worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolUtilCommands.java

```diff
@@ -122,4 +126,21 @@ public void size(Player player, LocalSession session,
         session.getBrushTool(player.getItemInHand(HandSide.MAIN_HAND).getType()).setSize(size);
         player.print("Brush size set.");
     }
+
+    @Command(
+            name = "tracemask",
+            desc = "Set the mask used to stop tool traces"
+    )
+    @CommandPermissions("worldedit.brush.options.tracemask")
+    public void traceMask(Player player, LocalSession session,
+                     @Arg(desc = "The mask to set", def = "")
+                             Mask mask) throws WorldEditException {
+        if (mask == null) {
+            session.getBrushTool(player.getItemInHand(HandSide.MAIN_HAND).getType()).setTraceMask(null);
+            player.print("Trace mask disabled.");
+        } else {
+            session.getBrushTool(player.getItemInHand(HandSide.MAIN_HAND).getType()).setTraceMask(mask);
```

Comment:
Wouldn't it be a better idea to move this line outside the conditional? It's essentially the same as the one before.